docs/Get Started/Set up Cognitive Services.ipynb

```diff
@@ -64,6 +64,149 @@
     "* [Create an Azure Cognitive multi-service (Decision, Language, Speech, Vision) resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesAllInOne). Alternatively, you can follow the steps to [create Single-service resource](https://learn.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account?tabs=decision%2Canomaly-detector%2Clanguage-service%2Ccomputer-vision%2Cwindows#create-a-new-azure-cognitive-services-resource). \n",
     "* Get your Cognitive Service resource's key. After your resource is successfully deployed, select **Next Steps** > **Go to resource**. Once at the resource, you can get the key from **Resource Management** > **Keys and Endpoint**. Copy the key and paste it into the notebook. Store keys securely and do not share them. "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Manage secrets with Azure Key Vault and access the secrets with find_secret"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After you create an Azure AI resource, you will obtain a resource key. You can use the resource key string to validate your subscription for a service, but it is highly recommended to manage it with Azure Key Vault. Azure Key Vault is a cloud-based service that allows you to store and manage cryptographic keys, certificates, and secrets used by cloud applications and services.\n",
+    "\n",
+    "You can skip the following content if you want to use a secret string instead of Azure Key Vault. (Do not recommend.)"
```

Comment:
```suggestion
    "You can skip the following content if you want to use a secret string instead of Azure Key Vault. This is not recommended for production workloads."
```